src/Platform/Microsoft.Testing.Platform/Logging/FileLoggerProvider.cs

```diff
@@ -29,11 +39,11 @@ public FileLoggerProvider(string logFolder, IClock clock, LogLevel logLevel, str
 
     public LogLevel LogLevel { get; }
 
-    public FileLogger FileLogger { get; private set; }
+    public FileLogger? FileLogger { get; private set; }
```

Comment:
Is it nullable for testing?

Reply:
Yes, made it nullable to avoid having to instantiate it. Should I revert it ?

Reply:
Well the problem is that with the change, all callers of this property now have to handle null check when we know this is not null.

Depending on whether the `FileLogger` ctor does something or not, I would either make the setter internal with the comment "for testing" or take `FileLogger` in the "for testing" ctor.

Reply:
Reverted.